graphs/adapter-api/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/api/GraphBehaviours.kt

```diff
@@ -29,3 +32,15 @@ open class AddEdgeBehaviour : ReifiedBehavior {
 open class GetShortestPathBehaviour : ReifiedBehavior {
     override val behavior = GraphBehaviours.GetShortestPath
 }
+
+open class GetCycleBehavior : ReifiedBehavior {
+    override val behavior = GraphBehaviours.HasCycle
+}
+
+open class GetCycleCountBehavior : ReifiedBehavior {
+    override val behavior = GraphBehaviours.GetNumberOfCycles
+}
```

Comment:
❓ Why the lack of consistency in the naming?

```suggestion
open class HasCycleBehavior : ReifiedBehavior {
    override val behavior = GraphBehaviours.HasCycle
}

open class GetNumberOfCyclesBehavior : ReifiedBehavior {
    override val behavior = GraphBehaviours.GetNumberOfCycles
}
```

P.S.: I don't mind whether you choose `GetNumberOfCycles` or `GetCycleCount` for the latter, as long as the name is consistent.